src/main/java/org/opensearch/security/rest/DashboardsInfoAction.java

```diff
@@ -59,7 +59,10 @@
 public class DashboardsInfoAction extends BaseRestHandler {
     private static final List<Route> routes = ImmutableList.of(
             new Route(GET, "/_opendistro/_security/kibanainfo"),
-            new Route(POST, "/_opendistro/_security/kibanainfo")
+            new Route(POST, "/_opendistro/_security/kibanainfo"),
+
+            new Route(GET, "/_plugins/_security/kibanainfo"),
```

Comment:
this should be `/_plugins/_security/dashboardsinfo`

Reply:
Updated both DashboardsInfoAction and unit tests